src/test/java/my/test/email/EmailMainTest.java

```diff
@@ -0,0 +1,43 @@
+package my.test.email;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+import org.openqa.selenium.WebDriver;
+
+public class EmailMainTest {
+    private LoginPage loginPage;
+    private LogoutPage logoutPage;
+
+    @BeforeEach
+    void setup() {
+        WebDriver driver = CreateDriver.getInstance().getDriver();
```

Comment:
Remove this line and remove WebDriver instance as parameter in constructors. Page object ctors will be without params and just put line inside it:
`this.driver = CreateDriver...`